Update hardware-software-requirements-2013.md (#3533)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/install/hardware-software-requirements-2013.md

> SQL Server 2014 requires the May 2014 Cumulative Update to be installed. To install the May 2014 Cumulative Update see [Updates to SharePoint 2013](/officeupdates/). 
  
> [!NOTE]

> > SQL Server 2016 or any newer version of SQL Server are not supported for SharePoint Server 2013. 

Reviewer: ```suggestion
> SQL Server 2016 or any later version of SQL Server is not supported on SharePoint Server 2013. 
```
